.changeset/tender-birds-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/palette': patch
```

Comment:
Not sure that we need a changeset but won't block on this!

Reply:
I think we need a changeset to make sure it publishes

Reply:
Duh you're right.. you'll have to add a changeset to your typography PR then as well

Reply:
Will do 👍